src-docs/src/views/auto_refresh/auto_refresh_input.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+import { EuiAutoRefresh, OnRefreshChangeProps } from '../../../../src';
+
+export default () => {
+  const [refreshInterval, setRefreshInterval] = useState(1000);
+  const [isPaused, setIsPaused] = useState(true);
+
+  const onRefreshChange = ({
+    isPaused,
+    refreshInterval,
+  }: OnRefreshChangeProps) => {
+    setIsPaused(isPaused);
+    setRefreshInterval(refreshInterval);
+  };
+
+  return (
+    <EuiAutoRefresh
+      isPaused={isPaused}
```

Comment:
It looks like this needs a form label because it's an input; CI is failing on axe checks without one.

```
Errors on tp://localhost:9999/#/forms/auto-refresh
08:53:28 [label]: Ensures every form element has a label
08:53:28   Help: https://dequeuniversity.com/rules/axe/4.1/label?application=axe-puppeteer
08:53:28   Elements:
08:53:28     - .euiFieldText
08:53:28 1 accessibility errors
```

I _think_ this should work, double-checking
```suggestion
    <EuiAutoRefresh
      aria-label="Auto refresh demo"
      isPaused={isPaused}
```

Reply:
Thanks for tracking that down. I've actually just added this directly to the Auto refresh input. https://github.com/elastic/eui/pull/5383/commits/422f3fd27b0dc0d9c054d915b1c5324dcf4d6943

Reply:
Nice, a much more elegant fix! 👍